### PR TITLE
Added Pebble support

### PR DESCRIPTION
**Pull request by ecmel**
Fixes issue https://github.com/micronaut-projects/micronaut-views/issues/127

### CODE REVIEW

**Comment by graemerocher** [comment]
Nice, don't worry about the failure, it is related to our test report publishing

File: views-pebble/src/main/resources/META-INF/native-image/io.micronaut.views.handlebars/native-image.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+Args = --initialize-at-build-time=com.mitchellbosecke.pebble
```
Comment (alvarosanchez) [inline views-pebble/src/main/resources/META-INF/native-image/io.micronaut.views.handlebars/native-image.properties:1]: File should be renamed

**Review by alvarosanchez — CHANGES_REQUESTED** [review]

File: views-pebble/src/main/resources/META-INF/native-image/io.micronaut.views.handlebars/native-image.properties (hunk 1 above)
Comment (ecmel) [reply views-pebble/src/main/resources/META-INF/native-image/io.micronaut.views.handlebars/native-image.properties]: Fixed

**Review by ilopmar — CHANGES_REQUESTED** [review]
As this is a new feature it should go in a next minor version. Please change the `@since` tags to 2.2.0.

**Comment by ilopmar** [comment]
@ecmel One question. Did you try GraalVM to test that it actually works? In that case it would be great to add a new test application to our official test-suite. Please take a look at the examples here: https://github.com/micronaut-graal-tests/micronaut-views-graal
It's a matter of creating a new branch, in this case it would be `2.3.x_pebble` based on any 2.2.x_... branch that already exist.

**Comment by ecmel** [comment]
@ilopmar I will add the test for graalvm.

On the other hand, Pebble has a parallel rendering [feature](https://pebbletemplates.io/wiki/guide/high-performance/) with an `ExecuterService`. Is it okey to add TaskExecuters.IO to the Builder? I am not sure if this feature is used a lot.

**Review by alvarosanchez — APPROVED** [review]

**Review by ilopmar — APPROVED** [review]

**Comment by ecmel** [comment]
@ilopmar Pebble's default loader does not work with graalvm native image. I will try to replace the loader.

**Comment by ecmel** [comment]
@ilopmar Added Pebble branch to https://github.com/micronaut-graal-tests/micronaut-views-graal

**Comment by ilopmar** [comment]
> @ilopmar Added Pebble branch to https://github.com/micronaut-graal-tests/micronaut-views-graal

Awesome! I'll take a look at that PR soon. Thanks :)

**Comment by ilopmar** [comment]
I've tested the GraalVM support and works. Unless @graemerocher or @alvarosanchez have any more comments on this PR I think we can merge it for now and release a new version to be included in Micronaut 2.3.
Then @ecmel if you want to add more features/improvements you can send new PRs.

**Comment by graemerocher** [comment]
Fine by me

**Comment by ecmel** [comment]
Thank you for merging this pr

**Pull request merged**